tests/tests/swfs/avm2/xml_has_property_via_in/Test.as

```diff
@@ -0,0 +1,38 @@
+package {
+    import flash.display.Sprite;
+
+    public class Test extends Sprite {}
+}
+
+var a = new XML('<item val="example"><a/></item>');
+trace("@val in a");
+trace("@val" in a);
+trace("");
+
+trace("val in a");
+trace("val" in a);
+trace("");
+
+trace("item in a");
+trace("item" in a);
+trace("");
+
+trace("@item in a");
+trace("@item" in a);
+trace("");
+
+trace("a in a");
+trace("a" in a);
+trace("");
+
+trace("@a in a");
+trace("@a" in a);
+trace("");
+
+trace("0 in a");
+trace(0 in a);
+trace("");
+
+trace("1 in a");
+trace(1 in a);
+trace("");
```

Comment:
Can you add a test for isPropertyEnumerable?

Reply:
This currently fails, but seems to work with your PR #15120.

Reply:
We can land #1520 first and use this as a test for it.